src/lib/components/SgtSvgIcon/SgtSvgIcon.vue

```diff
@@ -55,13 +59,13 @@ export default class SgtSvgIcon extends Vue {
   @Prop({ required: false }) private hoverIcon: string;
   @Prop({ required: false }) private tooltip: string;
   @Prop({ required: false, default: false }) private disabled: boolean;
+  @Prop({ required: false, default: false }) private disableClick: boolean;
 }
 </script>
 
 <style lang="scss">
-.icon-container {
+.sgt-icon-container {
   display: inline-block;
-  cursor: pointer;
 
   &.disabled {
```

Comment:
Styles for the 'disabled' class won't work now. Modify the styles to make it work.

Reply:
done